announcements/1.11.4.md

```diff
@@ -0,0 +1,18 @@
+### Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.11.4.
+
+Please see [component versions](https://anbox-cloud.io/docs/component-versions) for a list of updated components.
+
+#### New features & improvements
+
+None
+
+#### Bugs
+
+A bug in Anbox causes the random webview based application crashes when upgrading the webview.apk to [94.0.4606.80](https://chromereleases.googleblog.com/2021/10/chrome-for-android-update.html). For this reason, we downgrade the webview version to [93.0.4577.82](https://chromereleases.googleblog.com/2021/09/chrome-for-android-update.html) for stability and are planning to provide the fix in the Anbox Cloud 1.12 release.
```

Comment:
```suggestion
A bug in Anbox caused random webview-based applications to crash when upgrading the webview.apk to [94.0.4606.80](https://chromereleases.googleblog.com/2021/10/chrome-for-android-update.html). For this reason, we downgraded the webview version to [93.0.4577.82](https://chromereleases.googleblog.com/2021/09/chrome-for-android-update.html) for stability, and we are planning to provide a fix in the Anbox Cloud 1.12 release.
```

Reply:
Just to confirm: A bug in **Anbox** or a bug in **Anbox Cloud**?

Reply:
hmm, I think it's better to call out *Anbox* here than *Anbox Cloud* as it's caused by the lower stack. Maybe @morphis has a better idea.

Reply:
Lets call it the `Anbox runtime` here. Anbox Cloud is the entire stack whereas Anbox is one particular thing in the stack. Calling it `Anbox runtime` should make the distinction clear.

Reply:
+1